build_scripts/performance_test.py

```diff
@@ -0,0 +1,69 @@
+"""Koninklijke Philips N.V., 2019 - 2020. All rights reserved.
+file to run the performance test on similarity tool"""
+import os
+import sys
+import time
+import lipsum
+import pandas as pd
+from subprocess_calls import call_subprocess
+
+
+def create_input():
+    """ Function used to generate the input file to do the performance test"""
+    row_size = 5  # given row size = 10 will generate around 17,000 rows of data in the excel file generated.
+    data = []
+    for __ in range(row_size):
+        data.extend(lipsum.paras(150, True).split('.'))
+    dataf = pd.DataFrame(data, columns=['Steps'])
+    dataf.index.name = 'Uniq ID'
+    dataf.to_excel(os.path.join(os.path.abspath(os.path.join(__file__, os.pardir)), "input_data.xlsx"))
+    print("successfully created the input file")
+
+
+def validate_perfo(time_val, input_data):
+    """ Function to validate the performance of tool"""
+    if not time_val < 0.045 * int(input_data):
```

Comment:
Would it be worth reading 0.045 from a configuration file instead of hard-coding it? That way, we can dynamically change the value if needed without modifying the code.

Reply:
added it as command line input`python3 build_scripts/performance_test.py 0.045`